test/separate_compilation/serialization/buildAll.precomp

```diff
@@ -0,0 +1 @@
+testGenLib.precomp
```

Comment:
What is this doing? Just redirecting to use the precomp for the other file?

Reply:
It's a soft link, and I guess this is how it shows up on github. 🤷

Reply:
Oh, 🤦 of course.